src/project/controller/person-collection.ts

```diff
@@ -18,7 +19,13 @@ class ProjectPersonCollection extends Controller {
       href: ctx.request.body.href
     };
 
-    await projectService.addPersonToProject(params);
+    const person = await projectService.addPersonToProject(params);
+
+    await addUserPrivilege(
+      person.principalUri,
```

Comment:
hey @Evert, this is giving me an error. 
Wondering if I was on the right track. My intuition is, I should also be adding the `project.href` instead of the `person.href`.

Reply:
I think this section should go into the `projectService` instead, because it's all part of 'adding a person to a project'. There you should also not have this issue because you always have a principalUrl.